source/extensions/filters/network/dubbo_proxy/conn_manager.cc

```diff
@@ -121,11 +119,7 @@ void ConnectionManager::dispatch() {
   try {
     bool underflow = false;
     while (!underflow) {
-      Network::FilterStatus status = decoder_->onData(request_buffer_, underflow);
-      if (status == Network::FilterStatus::StopIteration) {
-        stopped_ = true;
-        break;
-      }
+      decoder_->onData(request_buffer_, underflow);
```

Comment:
Why doesn't care about the return value for onData here?

Reply:
At present, when Decoder processes data, it either finishes processing or needs to wait until the data is entered, so there is no stop.